tests/templates/kuttl/ad-user-info/10-install-opa.yaml.j2

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: test
+        labels:
+          opa.stackable.tech/bundle: "true"
+      data:
+        test.rego: |
+          package test
+
+          import data.stackable.opa.userinfo.v1 as userinfo
+
+          currentUserInfoByUsername := userinfo.userInfoByUsername(input.username)
+          currentUserInfoById := userinfo.userInfoById(input.id)
+      ---
+      apiVersion: opa.stackable.tech/v1alpha1
+      kind: OpaCluster
+      metadata:
+        name: test-opa
+      spec:
+        image:
+{% if test_scenario['values']['opa-latest'].find(",") > 0 %}
+          custom: "{{ test_scenario['values']['opa-latest'].split(',')[1] }}"
+          productVersion: "{{ test_scenario['values']['opa-latest'].split(',')[0] }}"
+{% else %}
+          productVersion: "{{ test_scenario['values']['opa-latest'] }}"
+{% endif %}
+          pullPolicy: IfNotPresent
+        clusterConfig:
+          userInfo:
+            backend:
+              experimentalActiveDirectory:
+                ldapServer: sble-addc.sble.test
+                baseDistinguishedName: DC=sble,DC=test
+                customAttributeMappings:
+                  country: c
+                kerberosSecretClassName: kerberos-ad
+                tls:
+                  verification:
+                    server:
+                      caCert:
+                        secretClass: tls-ad
+            cache: # optional, enabled by default
+              entryTimeToLive: 60s # optional, defaults to 60s
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+          vectorAggregatorConfigMapName: vector-aggregator-discovery
+{% endif %}
+        servers:
+          config:
+            logging:
+              enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+          podOverrides:
+            spec:
+              volumes:
+              - name: kerberos
+                ephemeral:
+                  volumeClaimTemplate:
+                    metadata:
+                      annotations:
+                        secrets.stackable.tech/scope: service=opa
```

Comment:
I'm a bit confused why you are using podOverrides here. Shouldn't the operator set it to this exact value?

Reply:
The answer is that I'm a dumbass who used an old test I wrote ages ago. Will fix (and retest).

Reply:
c493329960e69d2895f6fcce17ef89cf8a44590c

Reply:
Makes perfect sense, thanks!